src/main/drivers/flash_w25q128fv.c

```diff
@@ -115,45 +119,71 @@ w25q128fvState_t w25q128fvState = { 0 };
 static bool w25q128fv_waitForReady(flashDevice_t *fdevice);
 static void w25q128fv_waitForTimeout(flashDevice_t *fdevice);
 
-static void w25q128fv_setTimeout(flashDevice_t *fdevice, uint32_t timeoutMillis)
+MMFLASH_CODE static void w25q128fv_setTimeout(flashDevice_t *fdevice, timeMs_t timeoutMillis)
 {
-    uint32_t now = HAL_GetTick();
-    fdevice->timeoutAt = now + timeoutMillis;
+    timeMs_t nowMs = microsISR() / 1000;
+    fdevice->timeoutAt = nowMs + timeoutMillis;
 }
 
-static void w25q128fv_performOneByteCommand(flashDeviceIO_t *io, uint8_t command)
+MMFLASH_CODE static void w25q128fv_performOneByteCommand(flashDeviceIO_t *io, uint8_t command)
 {
+#if defined(USE_QUADSPI)
     QUADSPI_TypeDef *quadSpi = io->handle.quadSpi;
     quadSpiTransmit1LINE(quadSpi, command, 0, NULL, 0);
+#elif defined(USE_OCTOSPI)
+    OCTOSPI_TypeDef *octoSpi = io->handle.octoSpi;
+    octoSpiTransmit1LINE(octoSpi, command, 0, NULL, 0);
+#endif
+
 }
 
-static void w25q128fv_performCommandWithAddress(flashDeviceIO_t *io, uint8_t command, uint32_t address)
+MMFLASH_CODE static void w25q128fv_performCommandWithAddress(flashDeviceIO_t *io, uint8_t command, uint32_t address)
 {
+#if defined(USE_QUADSPI)
     QUADSPI_TypeDef *quadSpi = io->handle.quadSpi;
 
     quadSpiInstructionWithAddress1LINE(quadSpi, command, 0, address & 0xffffff, W25Q128FV_ADDRESS_BITS);
+#elif defined(USE_OCTOSPI)
+    OCTOSPI_TypeDef *octoSpi = io->handle.octoSpi;
+
+    octoSpiInstructionWithAddress1LINE(octoSpi, command, 0, address & 0xffffff, W25Q128FV_ADDRESS_BITS);
+#endif
 }
 
-static void w25q128fv_writeEnable(flashDevice_t *fdevice)
+MMFLASH_CODE static void w25q128fv_writeEnable(flashDevice_t *fdevice)
 {
     w25q128fv_performOneByteCommand(&fdevice->io, W25Q128FV_INSTRUCTION_WRITE_ENABLE);
 }
 
-static uint8_t w25q128fv_readRegister(flashDeviceIO_t *io, uint8_t command)
+MMFLASH_CODE static uint8_t w25q128fv_readRegister(flashDeviceIO_t *io, uint8_t command)
 {
+    uint8_t in[1] = { 0 };
+#ifdef USE_QUADSPI
```

Comment:
```suggestion
#if defined(USE_QUADSPI)
```
to match #elif style

Reply:
good catch, fixed.